src/SignalR/common/Shared/MessageBuffer.cs

```diff
@@ -68,6 +74,11 @@ public MessageBuffer(ConnectionContext connection, IHubProtocol protocol)
         _ = RunTimer();
     }
 
+    // TODO: pass in limits
```

Comment:
bug?

Reply:
Main issue for the feature is tracking this